### DISCUSSION TITLE
[Section 3.9 - Primeiro script] - Como utilizar a autenticação aws sso do awscli 2.0

### DISCUSSION THREAD

**Discussion by alebarrionovo** [chgasparoto/curso-aws-com-terraform#34] [Q&A]
o script ensina como pegar a configuração que aws cli, configura com accesskey e secretkey, o que eu preciso mudar no script para eu utilizar o aws sso que esta disponivel no awscli 2.0 e que funcionando.
```hcl
# https://registry.terraform.io/providers/hashicorp/aws/latest/docs
provider "aws" {
  region  = "us-east-1" # Brasil -> us-east-1
  profile = "tf015"
}
```

**Comment by chgasparoto** [comment]
Fala @alebarrionovo , não sei pois nunca precisei utilizar. Vou descobrir e te falo.

Edit:
Você tem que ter instalado uma versão >= 3.26.0 do AWS provider.
Neste link tem algumas instruções de como configurar https://github.com/hashicorp/terraform-provider-aws/issues/10851#issuecomment-769381748

Abraço.
